modules/screen-wallet-add-new/index.js

```diff
@@ -0,0 +1,270 @@
+    marginTop: 25, width: 120
+  },
+  getAccount: {
+    marginRight: 5, marginTop: 4
+  },
+  balanceText: {
+    fontSize: 20, fontWeight: "bold"
+  },
+  balanceTxt: {
+    color: "#7C7C7C", fontSize: 14, alignSelf: "flex-end", textAlign: "right", fontWeight: "bold"
+  },
+  walletBalance: {
+    display: "flex", flexDirection: "row", justifyContent: "space-between", marginBottom: 28, marginTop: 14, paddingHorizontal: 23
+  },
+  walletText: {
+    fontSize: 16
+  },
+
+  tabView: {
+    width: "50%",
+    height: 48,
+    backgroundColor: "#F1F1F1",
+    flexDirection: "row",
+    justifyContent: "space-around",
+    alignItems: "center",
+    borderRadius: 10,
+    paddingVertical: 6,
+    marginTop: 10
+  },
+  selectedTab: {
+    backgroundColor: "#FFF",
+    shadowColor: "gray",
+    elevation: 10
+  },
+  tabItem: {
+    height: "100%",
+    width: "45%",
+    paddingHorizontal: 10,
+    justifyContent: "center",
+    alignItems: "center",
+    backgroundColor: "#F1F1F1",
+    borderRadius: 10,
+    fontWeight: "bold"
+  },
+  bottomContainer: {
+    padding: 10
+  },
+  walletCard: { backgroundColor: "white", borderRadius: 10, width: "100%", padding: 10, display: "flex", flexDirection: "row", alignItems: "center", justifyContent: "space-between" },
+  myWallet: {
+    borderRadius: 10,
+    backgroundColor: "#FCF1D6",
+    alignItems: "center",
+    justifyContent: "center",
+    height: 80,
+    width: 80
+  },
+  walletName: {
+    color: "#26292A", fontSize: 14, marginLeft: 10, alignSelf: "center"
+  },
+  balanceContent: {
+    fontSize: 12,
+    color: "#7C7C7C",
+    alignSelf: "flex-end"
+
+  },
+  walletCardDetailContainer: {
+    marginTop: 17,
+    backgroundColor: "white",
+    borderRadius: 10,
+    padding: 10,
+    display: "flex",
+    flexDirection: "column"
+  },
+  walletCardDetail: {
+    height: 45,
+    width: "100%",
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    justifyContent: "space-between"
+  },
+  onlineText: {
+    color: "#12D790"
+  },
+  terms: {
+    backgroundColor: "#fff",
+    borderRadius: 10,
+    marginVertical: 15,
+    padding: 15
+  },
+  buttonContainer: {
+    flexDirection: "row",
+    justifyContent: "space-between"
+  },
+  cancelBtn: {
+    flex: 1,
+    margin: 12,
+    borderRadius: 10,
+    borderWidth: 1
+  },
+  connectBtn: {
+    flex: 1,
+    margin: 12
+  },
+  separator: {
+    height: 1,
+    backgroundColor: "#F0F2F7"
+  }
+});
+
+export default WalletAddNewScreen;
+const Button = (props) => {
+  return (
+    <TouchableHighlight onPress={props.onPress} disabled={props.disabled} underlayColor='#DDDDDD'>
+      <View style={[btnStyles.button, {
+        backgroundColor: props.backgroundColor ? props.backgroundColor : "#000000",
+        height: props.height ? props.height : 49
+      }]}>
+        <Text style={[btnStyles.text, { color: props.color ? props.color : "#ffffff" }]}>{props.children}</Text>
+      </View>
+    </TouchableHighlight>
```

Comment:
Is it possible to convert the TouchableHighlight into a [Pressable](https://reactnative.dev/docs/pressable)?

Reply:
It is done.